src/main/java/no/nav/tag/kontaktskjema/SwaggerConfig.java

```diff
@@ -14,7 +14,7 @@
 
 @Configuration
 @EnableSwagger2
-@Profile({"preprod", "dev"})
+@Profile({"preprod", "dev", "local"})
```

Comment:
Tanken var av "dev" var local; hvis navneendring kan du fjerne dev :)

Reply:
Ja, men det er ikke helt det samme. _"dev"_ er delt av alle utviklere siden den bruker application.properties som er sjekket inn i koden. Med en egen _local_ kan man ha en egen ikke-innsjekket application-local.properties som definerer det lokale kjøretidsmiljøet (som ikke nødvendigvis er det samme oppsettet som når man kjører enhetstester).

Reply:
...eller sagt på en annen måte: Tanken var at bruk av _local_ er helt valgfritt, og bakoverkompatibelt. Ved å beholde _dev_ her kan alle gjøre som før, og ting vil fortsatt funke som før. Hvis vi fjerner _dev_ er alle nødt til å bruke profil _local_ for å få frem swagger.

Reply:
Good point!